prow/deploy.sh

```diff
@@ -88,4 +88,6 @@ for s in {5..1}; do
     sleep 1s
 done
 bazel run //prow/cluster:production.apply --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
```

Comment:
Can we please make this k8s_objects() target include //prow/monitoring:produciton.apply and //prow/monitoring/mixings/dashboards_out:grafana-configmaps.apply?

We should be able to update prow with a single call, not 3.

Reply:
I need more hints to merge the targets. If I understand it correctly, `//prow/cluster:production` accepts only `components`. However,

* some of objects use `k8s_configmap` function to generate the `k8s_object`.
* `components` only works with yaml files on the current folder.

Instead, use `multirun` to make a bit nicer.

https://github.com/kubernetes/test-infra/blob/c8298ea38c506a8c9d1688ffd0370b95551dd969/prow/cluster/BUILD.bazel#L47-L55